src/ui/public/agg_response/hierarchical/_tooltip.html

```diff
@@ -1,8 +1,16 @@
 <table class="visTooltip__table">
   <thead>
     <tr class="eui-textLeft visTooltip__label">
-      <th scope="col">field</th>
-      <th scope="col">value</th>
+      <th
+        scope="col"
+        i18n-id="common.ui.aggResponse.fieldLabel"
+        i18n-default-message="field"
```

Comment:
@timroes  We can't find this labels(field and value) on UI. Should we leave this labels translated, or revert this changes. Or can you help find this label on UI, or tell who can help?